[2.7] bpo-32784: Wrong argument name for csv.DictReader in documentation

matrixise opened the pull request:
Fix the documentation of csv.DictReader, csv.DictWriter, the name of the parameter for the file object does  not match the declared name in the __init__ method.

<!-- issue-number: bpo-32784 -->
https://bugs.python.org/issue32784
<!-- /issue-number -->

serhiy-storchaka reviewed (changes requested):
A NEWS entry is not needed for such trivial fix.

There are other references to csvfile in the documentation. Just backport the original changes if you want to made them in 2.7.

matrixise committed:
bpo-32784: Wrong argument name for csv.DictReader in documentation

Use f as the name of the first parameter of csv.DictReader and
csv.DictWriter classes.
